myia/abstract/data.py

```diff
@@ -595,23 +636,45 @@ def format_abstract(a):
 
 def pretty_type(t):
     """Pretty print a type."""
-    if dtype.ismyiatype(t, dtype.Float):
-        return f'f{t.bits}'
-    elif dtype.ismyiatype(t, dtype.Int):
-        return f'i{t.bits}'
-    elif dtype.ismyiatype(t, dtype.UInt):
-        return f'u{t.bits}'
+    if not isinstance(t, type):
+        return str(t)
+    elif issubclass(t, dtype.Float):
+        return f'f{getattr(t, "bits", "??")}'
+    elif issubclass(t, dtype.Int):
+        return f'i{getattr(t, "bits", "??")}'
+    elif issubclass(t, dtype.UInt):
+        return f'u{getattr(t, "bits", "??")}'
```

Comment:
The purpose of this is basically to print `i64` instead of `Int[64]` etc. Bool will just print `Bool`. Honestly, printing `Int[64]` isn't that bad, so maybe we don't want to have that function at all.